### PR TITLE
Fix empty array error

### PR DESCRIPTION
**Pull request by AndreEtienne**
This PR Fixes the problem that when the From is sent and there is an empty field (Node) in the array, it does not come to an error.

### CODE REVIEW

File: Classes/CRON/FormBuilder/Controller/FormBuilderController.php [hunk 1]
```diff
@@ -245,7 +246,9 @@ protected function sendMail($fields, $files)
             }
 
             $customerFields = array_filter($fields , function( $v ) {
-                return $v['node']->getProperty('filter') != true;
+                if (!empty($field)) {
```
Comment (baschny) [inline Classes/CRON/FormBuilder/Controller/FormBuilderController.php:251]: isn't it called `$v` here instead of `$field`?

Comment (baschny) [reply Classes/CRON/FormBuilder/Controller/FormBuilderController.php:251]: maybe rename `$v` to `$field` in the anonymous function's parameter to be more precise and understandable?

File: Classes/CRON/FormBuilder/Controller/FormBuilderController.php [hunk 2]
```diff
@@ -230,13 +230,14 @@ protected function sendMail($fields, $files)
...
 
             foreach ($fields as $field) {
-
-                if ($field['node']->getProperty('type') == "email" && $field['node']->getProperty('isCustomerMail')) {
-                    $customerMail = $field['value'];
-                }
-
-                if ($field['node']->getProperty('type') == "name") {
-                    $customerName = $field['value'];
+                if (!empty($field)) {
```
Comment (baschny) [inline Classes/CRON/FormBuilder/Controller/FormBuilderController.php]: i prefer the "guard-condition" style:

```
            foreach ($fields as $field) {
                if (empty($field)) {
                    continue;
                }

                ...
            }
```

as it makes the body of the loop less indented and easier to follow. But its a matter of taste.

**Review by baschny — CHANGES_REQUESTED** [review]
See comments

File: Classes/CRON/FormBuilder/Controller/FormBuilderController.php (hunk 1 above)
Comment (AndreEtienne) [reply Classes/CRON/FormBuilder/Controller/FormBuilderController.php:251]: Yes, you are right. I have renamed the variable.

File: Classes/CRON/FormBuilder/Controller/FormBuilderController.php (hunk 2 above)
Comment (AndreEtienne) [reply Classes/CRON/FormBuilder/Controller/FormBuilderController.php]: I also find more beautiful I will remember.

**Pull request merged**